R/dplyr.R

```diff
@@ -220,16 +220,6 @@ mutate.spark_tbl <- function(.data, ...) {
   new_spark_tbl(sdf, groups = attr(.data, "groups"))
 }
 
-#' @export
-coalesce <- function(...) {
-  UseMethod("coalesce")
-}
-
-#' @importFrom dplyr coalesce
-coalesce.default <- function(...) {
-  dplyr::coalesce(...)
-}
-
```

Comment:
Got to undo this

Reply:
Yes will do, I didn't realize they were needed in both places.